src/python/genny/genny_auto_tasks.py

```diff
@@ -102,20 +117,33 @@ def main():
         description="Generates json that can be used as input to evergreen's generate.tasks, representing genny workloads to be run")
 
     parser.add_argument('--variants', nargs='+', required=True, help='buildvariants that workloads should run on')
-    parser.add_argument('-o', '--output', default='build/generated_tasks.json')
+    parser.add_argument('--workloads', nargs='+', help='paths of workloads to run, relative to genny/src/workloads/')
```

Comment:
``relative to `src/workloads` in the genny repository root`` because we don't know that genny is checked out into a dir named `genny` 😸

Reply:
Done.